source/android/adaptivecards/src/test/java/io/adaptivecards/ExampleUnitTest.java

```diff
@@ -1,17 +0,0 @@
-package io.adaptivecards;
-
-import org.junit.Test;
-
-import static org.junit.Assert.*;
-
-/**
```

Comment:
FYI that we will be adding unit tests soon; @almedina-ms FYI

Reply:
Awesome! Would love to contribute.